client/views/popup.js

```diff
@@ -119,6 +124,12 @@ class Popup {
         const availWidthLeft = box.left - viewport.right - 3;
         const availWidthRight = viewport.right - box.left - 3;
 
+        [...Popup.els]
```

Comment:
Not sure I understand what happens here...

Reply:
Yep, that was experiments. Forgot to remove.